### PR TITLE
TW-445 doc import API from AWS gateway

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+* [Supported API schema formats](#supported-api-schema-formats)
+
+## Importing API schemas
+
+To import your API specifications into Postman:
+
+1. From the sidebar, select __APIs__.
+1. Select **Import** in the left navigation menu.
+1. Select a file or folder, input a link to the API, or paste your raw text.
+    > You can also import from a code repository or API gateway. See below for details.
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/importing-an-api.md:38]: Add blank lines above and below note

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+You will receive a confirmation once the import has completed. You can now view your newly imported API schemas and generated collections in Postman.
+
+## Importing API schemas from Bitbucket
+
+> You must sign in to a [Postman account](/docs/getting-started/postman-account/#signing-up-for-a-postman-account) to use this feature.
+
+You can import API schemas from a Bitbucket repository.
+
+1. Select **Import** > **Code repository** > **Bitbucket**.
```
Comment (mark-anthony-postman) [inline src/pages/docs/designing-and-developing-your-api/importing-an-api.md:74]: Same comment as above. Maybe use:
1. From the sidebar, select **APIs** and then select **Import**.
2. Select Bitbucket.

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+1. Postman shows a preview of the API schemas to import into your workspace. Select **Import** to confirm and complete the import.
+
+You will receive a confirmation once the import has completed. You can now view your newly imported API schemas and generated collections in Postman.
+
+## Importing API schemas from Amazon API Gateway
+
+[Amazon API Gateway](https://aws.amazon.com/api-gateway/) enables developers to publish and manage APIs that access AWS as well as other web services and data sources. You can import APIs directly from AWS Gateway to Postman. This creates an integration with API Gateway. Once connected, you can view your API deployment status and history from within Postman. You can also view [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) metrics for each stage to get operational insight into your API.
+
+This integration supports importing OpenAPI 3.0 schemas for both HTTP and REST APIs. Importing will create a new API in Postman, with the API name set as the name of the API Gateway, and the default API version being the name of stage/environment. The new API will also have the AWS API Gateway integration added to it.
```
Comment (mark-anthony-postman) [inline src/pages/docs/designing-and-developing-your-api/importing-an-api.md:88]:
```suggestion
This integration supports importing OpenAPI 3.0 schemas for both HTTP and REST APIs. Importing will create a new API in Postman, with the API name set as the name of the API Gateway, and the default API version set as the name of the stage or environment. The new API will also have the AWS API Gateway integration added to it.
```

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+To import an API from AWS API Gateway:
+
+1. From the sidebar, select __APIs__.
```
Comment (mark-anthony-postman) [inline src/pages/docs/designing-and-developing-your-api/importing-an-api.md:92]: Again, I'd suggest:
"From the sidebar, select **APIs** and then select **Import**."

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+    }
+    ```
+
+7. Select **Next: Tags**.
+8. Select **Next: Review**.
+9. Add a **Role name** and **Role description**, then select **Create role**.
+
+Copy the **Role ARN** from AWS and paste it in Postman under **Step 2: Enter role ARN and region**. Next, enter the **AWS Region** where the API Gateway is located and select the **API Gateway**. Then, enter an **API Name** for the imported API. When you're ready, select **Connect**.
+
+#### Updating an exiting IAM policy for CloudWatch
```
Comment (mark-anthony-postman) [inline src/pages/docs/designing-and-developing-your-api/importing-an-api.md:146]: I think you can delete this section. They wouldn't have an old integration laying around, as this import feature is coming out after CloudWatch was released.

**Review by mark-anthony-postman — APPROVED** [review]
Wow, nice job knocking this out! 😄  Made a few suggestions but otherwise LGTM! 🌟

File: src/pages/docs/designing-and-developing-your-api/importing-an-api.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/designing-and-developing-your-api/importing-an-api.md]: I added the blank lines, but you added an extra four spaces to the front of the note, which breaks lint with `MD046 - Code block style` (?!)

**Review by vinitshahdeo — APPROVED** [review]

**Pull request merged**